docs/spec/v2alpha1/helmreleases.md

```diff
@@ -142,6 +142,11 @@ type Install struct {
 	// +optional
```

Comment:
This document is kept for historical reason, but should not include any changes as the `v2alpha1` has never supported these fields. Instead, they should be documented in `v2beta1`.

Reply:
Done!